vms/volunteer/models.py

```diff
@@ -1,100 +1,101 @@
-# Django

```

Comment:
Yes, I'm using pycharm. I pointed this out in the vms channel too that setup config is showing to use LF but actually the repo is using CRLF

Reply:
https://github.com/systers/vms/blob/develop/.editorconfig#L7
Are you talking about this?
Lemme see.
How do you the repo is using CRLF?

Reply:
Because when I change the line ending locally in pycharm then make a push then only my changes will be shown, not as a whole file.

Reply:
Ohh okay. Can something be done regarding this? You know some fix?

Reply:
I'll have to look into some auto-fix things otherwise the final option would be to go through every file and make it LF, prolly we can raise an issue for it now and discuss it tomorrow in meeting?

Reply:
Lets discuss it on slack as well as in meeting today.

Reply:
https://help.github.com/articles/dealing-with-line-endings Did you try this?

Reply:
Then maybe lets not address this issue as of now and deal with this in another issue. What do you say @naman1901 @Monal5031.